pwndbg/commands/hexdump.py

```diff
@@ -32,13 +33,17 @@
 def hexdump(address=None, count=pwndbg.config.hexdump_bytes):
     address = int(address)
     address &= pwndbg.arch.ptrmask
-    count   = int(count)
+    count   = max(int(count), 0)
```

Comment:
it still works like that as the address won't be negative even when the "binary representation" of a signed integer type would be a negative number, it just ansures. it will just subtract the value later.
the only behavior that triggers this was passing -10 as a second argument, which breaks this command.
so all fine :smile: